server/index.js

```diff
@@ -1,8 +1,9 @@
 import { ApolloServer } from 'apollo-server-express'
 import { importSchema } from 'graphql-import'
+import { promisify } from 'util'
```

Comment:
What's the difference of using promisify vs async-redis?

Reply:
If you noticed, I switched dependendencies, from `async-redis` to `redis`. The latter one is the official one suggested in the actual Redis docs. The  only drawback it has, it doesn't support yet `async/await` asynchronous calls out of the box.

`promisify` is a built-in Node.js dependency to wrap the `redis` operations in an async manner. Thus allowing us to do something like:
```
const article = await cache.get(id);
```
rather than:
```
cache.get(id, callback => {...})
```

So actually, `async-redis` its not being substituted by `promisify`, but rather by `redis`. `promisify` just allows us to have that "syntactic sugar" available for the cache.